Patches/Alpha Animals/ThingDefs_Misc/AlphaAnimals_CE_Patch_RangedMechanoid.xml

```diff
@@ -89,9 +89,9 @@
 			  <defaultProjectile>Bullet_Flamethrower_Prometheum</defaultProjectile>
 			  <warmupTime>0.15</warmupTime>
 			  <range>16</range>
-			  <ticksBetweenBurstShots>0.5</ticksBetweenBurstShots>
+			  <ticksBetweenBurstShots>1</ticksBetweenBurstShots>
 			  <burstShotCount>5</burstShotCount>
-			  <soundCast>HissFlamethrower</soundCast>
+			  <soundCast>ThrowMolotovCocktail</soundCast>

```

Comment:
Why change this?

Reply:
Rimworld creates a warning that HissFlamethrower is not found. Alpha Animals originally used ThrowMolotovCocktail, so i just changed it back.